### PR TITLE
Update policy-csp-experience.md in regards to Chat icon policy

### PR DESCRIPTION
**Pull request by harshparakh**

### CODE REVIEW

File: windows/client-management/mdm/policy-csp-experience.md [hunk 1]
```diff
@@ -1234,6 +1234,8 @@ The values for this policy are 0, 1, 2, and 3. This policy defaults to 0 if not
 -   2 - Hide: The Chat icon will be hidden by default. Users can show or hide it in Settings.
 -   3 - Disabled: The Chat icon will not be displayed, and users cannot show or hide it in Settings.
 
+Note: Options 1 and 2 (Show, Hide) only work on first boot whereas option 3 - Disabled will work on all attempts.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-experience.md:1237]:
```suggestion
> [!NOTE]
> Options 1 and 2 (Show, Hide) only work on the first sign-in attempt whereas Option 3 (Disabled) works on all attempts.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @harshparakh, kindly commit the suggested changes if you agree with them. Many thanks.

File: windows/client-management/mdm/policy-csp-experience.md [hunk 2]
```diff
@@ -1234,6 +1234,9 @@ The values for this policy are 0, 1, 2, and 3. This policy defaults to 0 if not
 -   2 - Hide: The Chat icon will be hidden by default. Users can show or hide it in Settings.
 -   3 - Disabled: The Chat icon will not be displayed, and users cannot show or hide it in Settings.
 
+> [!NOTE]
+> Options 1 and 2 (Show, Hide) only work on the first sign-in attempt whereas Option 3 (Disabled) works on all attempts.
```
Comment (harshparakh) [inline windows/client-management/mdm/policy-csp-experience.md:1238]:
```suggestion
> Option 1 (Show) and Option 2 (Hide) only work on the first sign-in attempt. Option 3 (Disabled) works on all attempts.
```

**Comment by garycentric** [comment]
@harshparakh 
Microsoft-credentialed contributors are strongly encouraged to work in the private repos, not the public. The private repos provide checks for build integrity, checks for spelling & grammar, and provide preview links for added and modified articles. The private repos also have a shorter path to publication, and they're a secure place to add content prior to official release. 

For more information about working in the private repos, please see [Where do I start?](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=master#where-do-i-start) in the MARVEL authoring guide. If you have questions or need help, please contact [Ask an Admin](https://aka.ms/askanadmin) in Teams.

**Comment by harshparakh** [comment]
> @harshparakh Microsoft-credentialed contributors are strongly encouraged to work in the private repos, not the public. The private repos provide checks for build integrity, checks for spelling & grammar, and provide preview links for added and modified articles. The private repos also have a shorter path to publication, and they're a secure place to add content prior to official release.
> 
> For more information about working in the private repos, please see [Where do I start?](https://review.docs.microsoft.com/en-us/office-authoring-guide/office-docs-authoring-overview?branch=master#where-do-i-start) in the MARVEL authoring guide. If you have questions or need help, please contact [Ask an Admin](https://aka.ms/askanadmin) in Teams.

Thanks @garycentric, that makes sense. @Dansimp is helping me out with access to the private repository.

**Pull request merged**